src/wui/http/http_client.c

```diff
@@ -448,7 +449,7 @@ static const char * telemetry_to_http_str(const char * host_ip4_str){
 
     eeprom_get_string(EEVAR_CONNECT_KEY_START, printer_token, API_TOKEN_LEN);
     printer_token[API_TOKEN_LEN] = 0;
-    snprintf(header, HEADER_MAX_SIZE, "POST %s HTTP/1.0\r\nHost: %s\nPrinter-Token: %s\r\n", uri, host_ip4_str, printer_token);
+    snprintf(header, HEADER_MAX_SIZE, "POST %s HTTP/1.0\r\nHost: %s\nPrinter-Token: %s\r\n\r\n", uri, host_ip4_str, printer_token);
```

Comment:
we have to combine the header generation used in HTTP server replies

Reply:
I have already done that but in different branch: 
https://github.com/michalrudolf/Prusa-Firmware-Buddy/tree/A3IDES-687_STATE_CHANGED